_config/config.yml

```diff
@@ -1,6 +1,12 @@
 LeftAndMain:
   extensions:
     - LeftAndMainPageIconsExtension
+ContentController:
```

Comment:
I wonder if we should just go ahead and promote `ErrorPageControllerExtension` onto `Controller`?

That would cover Form::httpError, etc.